dev.lasheen.qr.yml

```diff
@@ -0,0 +1,28 @@
+app-id: dev.lasheen.qr
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: qr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  - name: qr
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - "./build-flatpak.sh"
+    sources:
+      - type: file
+        url: https://github.com/YoussefLasheen/qr/releases/download/v1.0.0/qr-Linux-Portable.tar.gz
```

Comment:
you still build a flatpak for aarch64 with x86_64 binaries

Reply:
I added a flathub.json file to build for x86_64 only